### DISCUSSION TITLE
Hot Corners/ Hot edges in cinnamon

### DISCUSSION THREAD

**Discussion by jacquesbg** [linuxmint/discussions#222] [Questions] [answered]
How can I switch off the hot corner behaviour?

Even when hot corners are disabled, they still work, see the screenshot below.
![hotcorners](https://github.com/linuxmint/discussions/assets/52759960/6ef301e7-e1ea-4112-a49b-c6575be7de35)

PS:
I tried to uninstall hot corners through synaptic, but it turns out that uninstalling hot corners requires to unisntall cinnamon-dbg, and then will uninstall cinnamon as a whole?

PS2:
 In gnome there is a tiling assisstant (gome-extension) whih allows the use of super (windows keys).
   - super (=windows-key)-left moves window to the left half of the screen
   - super-up to top half
   - etc

 I prefer this much above the automatic hot corner behaviour

**Comment by rcalixte** [comment, marked as answer]
1. This isn't the Hot Corners setting but instead Window Tiling.
Once you turn it off, the tiling when you drag a window to the border of the screen will be disabled as well.

![2023-11-20_15-49-47](https://github.com/linuxmint/discussions/assets/10281587/a7378825-cc21-419d-a375-4202f71c872c)



2. Under Keyboard -> Shortcuts -> Windows, there are options for Positioning and Tiling and Snapping where you can set your preferred bindings.

**Reply by jacquesbg** [reply]
Thanks, that is exactly the answer I needed to resolve this!
